dll/win32/shlwapi/utils.cpp

```diff
@@ -6,13 +6,31 @@
  */
 
 #define _ATL_NO_EXCEPTIONS
+
+/*
+ * HACK!  These functions are conflicting with <shobjidl.h> inline functions...
```

Comment:
(Extra space after the "HACK! " and the text, you may remove it here and in the other instances)

Reply:
Extra spaces deleted.